src/components/NestedAccordionMenu/MenuItem.js

```diff
@@ -0,0 +1,102 @@
+import React, { Fragment, useState } from "react";
+import List from "@material-ui/core/List";
+import ListItem from "@material-ui/core/ListItem";
+import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
+import ExpandLessIcon from "@material-ui/icons/ExpandLess";
+import Collapse from "@material-ui/core/Collapse";
+import ListItemIcon from "@material-ui/core/ListItemIcon";
+import ListItemText from "@material-ui/core/ListItemText";
+import { makeStyles } from "@material-ui/core/styles";
+const useStyles = makeStyles(theme => ({
```

Comment:
Extract styles to a styles.js file. That means that you also need to make MenuItems a folder (thus with index.js and style.js)